components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -70,39 +118,67 @@ export type DrawerNavGroupProps = {
     chevron?: boolean;
     content?: ReactNode;
     divider?: boolean;
+    nestedDivider?: boolean;
     fontColor?: string;
     iconColor?: string;
     items: NavItem[];
     onSelect?: Function;
     open?: boolean;
+    ripple?: boolean;
     title?: string;
     titleColor?: string;
+    hidePadding?: boolean;
```

Comment:
Does this need to be exposed as a prop, or something that we want to control internally?

Reply:
Resolve: the user need to have access to this prop in order to hide the padding when there's no icons